sources/postgres/README.md

```diff
@@ -1,83 +1,92 @@
+schema mapping from source to destination. You will need to specify this file
+during data migration. You can also specify a particular Spanner instance to use
+during data migration.
 
-To specify a particular Spanner instance to use, run:
+For example, run
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump -instance my-spanner-instance
+harbourbridge data -session=mydb.session.json -source=pg -target-profile="instance=my-spanner-instance" < my_pg_dump_file
 ```
 
 By default, HarbourBridge will generate a new Spanner database name to populate.
 You can override this and specify the database name to use by:
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump -dbname my-spanner-database-name
+harbourbridge data -session=mydb.session.json -source=pg -target-profile="instance=my-spanner-instance,dbname=my-spanner-database-name" < my_pg_dump_file
+```
+
+You can also run HarbourBridge in a oneshot mode, where it will perform both
+schema and data migration. This is useful for quick evaluation when source
+database size is small.
+
+```sh
+harbourbridge oneshot -source=pg -target-profile="instance=my-spanner-instance" < my_pg_dump_file
 ```
 
 HarbourBridge generates a report file, a schema file, and a bad-data file (if
 there are bad-data rows). You can control where these files are written by
 specifying a file prefix. For example,
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump -prefix mydb.
+harbourbridge schema -prefix=mydb. -source=postgres < my_pg_dump_file
 ```
 
 will write files `mydb.report.txt`, `mydb.schema.txt`, and
 `mydb.dropped.txt`. The prefix can also be a directory. For example,
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump -prefix ~/spanner-eval-mydb/
+harbourbridge schema -prefix=~/spanner-eval-mydb/ -source=postgres < my_pg_dump_file
 ```
 
 would write the files into the directory `~/spanner-eval-mydb/`. Note
 that HarbourBridge will not create directories as it writes these files.
 
 ### Directly connecting to a PostgreSQL database
 
-To use the tool directly on a PostgresSQL database called mydb, run
+In this case, HarbourBridge connects directly to the PostgreSQL database to
+retrieve table schema and data. Set the `-source=postgres` and corresponding
+source profile connection parameters `host`, `port`, `user`, `db_name` and
+`password`.
+
+For example to perform schema conversion, run
 
 ```sh
-harbourbridge -driver=postgres
+harbourbridge schema -source=postgres -source-profile="host=<>,port=<>,user=<>,db_name=<>"
 ```
 
-It is assumed that _PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_ environment
-variables are set. Password can be specified either in the _PGPASSWORD_ environment
-variable or provided at the password prompt.
+Parameters `port` and `password` are optional. Port (`port`) defaults to `5432`
+for PostgreSQL source. Password can be provided at the password prompt.
+
+(⚠ Deprecated ⚠) Set environment variables `PGHOST`, `PGPORT`, `PGUSER`,
```

Comment:
Suggest not deprecating the environment variables for postgres -- these are standard environment variables commonly used by postgres users.

In contrast, the mysql ones are non-standard harbourbridge-specific, so should be deprecated I think.

Reply:
SGTM, have modified the language to say "Alternatively" instead of "Deprecated".